python/pyspark/sql/functions/builtin.py

```diff
@@ -72,7 +72,7 @@
         UserDefinedFunctionLike,
     )
 
-if _has_numpy:  # type: ignore[has-type]
+if _has_numpy:
```

Comment:
Oh, this seems to make another issue.
```
starting mypy examples test...
examples failed mypy checks:
python/pyspark/sql/functions/builtin.py:75: error: Cannot determine type of "has_numpy"  [has-type]
Found 1 error in 1 file (checked 101 source files)
```

Reply:
hmmm .. this is weird. Let me add it back.